app/overrides/o_data_controller_override.rb

```diff
@@ -100,4 +102,13 @@ def entity.plural_name
       name
     end
   end
+
+  # Performs an action in the given locale, resetting to the original locale after finishing.
+  def in_locale(locale)
```

Comment:
Note: duplicated method because this code is run inside of the odata engine, not nemo.